aep/general/0146/aep.md.j2

```diff
@@ -86,9 +87,63 @@ which is an often-unfamiliar process.
 Because of this, `Any` **should not** be used unless other options are
 infeasible.
 
-### Generic fields in OAS APIs
-
-**Note:** OAS-specific guidance not yet written.
+{% tab oas %}
+
+#### Type Union
+
+Under the definition of the OpenAPI Specification, [JSONSchema][] keywords for
+schema composition (`allOf`, `anyOf`, `oneOf`, `not`) **may** be used to
+introduce a type union. The type union produced is equivalent to that of using
+`oneof` in protobuf APIs. Due to the schema's complexity, validation for this
+schema **should** be handled server-side, and it is fine to not document the
+schema explicitly.
```

Comment:
```suggestion
`oneof` in protobuf APIs. Due to the schema's complexity, validation for this
schema **should** be handled server-side, and OpenAPI schemas do not need to document the
schema explicitly.
```

"it is fine" feels a bit colloquial here - I'd say something along the lines of "do not need" or "are not required".